code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -17,13 +17,15 @@
 	icon = 'icons/obj/food/containers.dmi'
 	resistance_flags = FLAMMABLE
 	var/icon_type
+	/// Do we let the fancy subtype handle the examine?
+	var/special_examine = TRUE
```

Comment:
this variable is confusingly named, i think a proc would be better for getting the examine next for fancy boxes